Rocket.Chat/Models/Subscription.swift

```diff
@@ -159,4 +159,18 @@ extension Subscription {
         return object
     }
 
+    static func find(name: String, subscriptionType: [SubscriptionType], realm: Realm) -> Subscription? {
+        let predicate = NSPredicate(
+            format: "name == %@ && privateType IN %@",
+            name, subscriptionType.map { $0.rawValue }
+        )
+
+        return Realm.shared?.objects(Subscription.self).filter(predicate).first
```

Comment:
Why two methods if you're using Realm.shared? in both?

Reply:
Oops, I was sleepy when I was programming. Sorry 😶
I create a method that pass `Realm.shared` as default, to don't need import Realm in `ChatControllerMessageCellProtocol.swift`